homework37/src/main/java/com/vadim/tkach/lesson37/NumberUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.vadim.tkach.lesson37;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NumberUtils {
+    public static int getsum(List<Number> numbers) {
+        int sum = 0;
+        for (Number number : numbers) {
+                sum += number.intValue();
+        }
+        return sum;
+    }
+    public static List<Integer> sumOddNumbers(List<Number> numbers) {
+        List<Integer> result = new ArrayList<>();
+        for (Number number : numbers) {
+            int value = number.intValue();
+            if (value % 2 != 0) {
+              result.add(value * 2);
+            }
+        }
+        return result;
+    }
```

Comment:
Краще використати `long` замість `int` :) В такому випадку Ваш метод зможе корректно опрацювати масив з інтів та лонгів.

```suggestion
    public static List<Long> multiplyOddNumber(List<Number> numbers) {
        List<Long> result = new ArrayList<>();
        for (Number num : numbers) {
            long value = num.longValue();
            if (value % 2 !=0) {
                result.add(value * 2);
            }
        }
        return result;
    }
```